action.yml

```diff
@@ -80,9 +80,9 @@ runs:
             URL="https://pkgs.tailscale.com/unstable/tailscale_${VERSION}_${TS_ARCH}.tgz"
           fi
           if ! [[ "$SHA256SUM" ]] ; then
-            SHA256SUM="$(curl "${URL}.sha256")"
+            SHA256SUM="$(curl -L "${URL}.sha256")"
           fi
-          curl "$URL" -o tailscale.tgz --max-time 300
+          curl -L "$URL" -o tailscale.tgz --max-time 300
```

Comment:
set the user-agent too so we know we can serve from CDN?

Reply:
Done, and also fixed order.